future/dryrun.go

```diff
@@ -154,14 +154,11 @@ type DryrunResponse struct {
 }
 
 func NewDryrunResponse(d models.DryrunResponse) (DryrunResponse, error) {
-	// TODO:  this is lazy but also works?
-	dr := DryrunResponse{}
-	b, err := json.Marshal(dr)
+	b, err := json.Marshal(d)
```

Comment:
Is this right? We should have a comment that explains why we're doing this thing.
```suggestion
	// Marshal and unmarshal to fix integer types.
	b, err := json.Marshal(d)
```